components/popular-stories-cf/server.jsx

```diff
@@ -44,7 +44,7 @@ export default async (args, info) => {
                       ]
                   }
                   orderBy: [count_DESC]
-                  limit: 5
+                  limit: 10
```

Comment:
@dlcousins any reason why this is 10? Does this help with making sure there are always enough items in the popular stories if for say one of the 5 need to be excluded? (guessing)